common/djangoapps/third_party_auth/tests/test_middleware.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tests for third party auth middleware
+"""
+import mock
+from third_party_auth.tests.testutil import TestCase
+from third_party_auth.middleware import ExceptionMiddleware
+from requests.exceptions import HTTPError
+from django.test.client import RequestFactory
+from django.contrib.messages.middleware import MessageMiddleware
+
+
+class ThirdPartyAuthMiddlewareTestCase(TestCase):
+
+    @mock.patch('django.conf.settings.MESSAGE_STORAGE', 'django.contrib.messages.storage.cookie.CookieStorage')
+    def test_http_exception_redirection(self):
+        """
+        Test ExceptionMiddleware is correctly redirected to login page
+        when PSA raises HttpError exception.
+        """
+
+        test_url = '/login'
```

Comment:
nit: better if variable name is something like `login_url`.

Reply:
better if use `reverse('login')`

Reply:
I still believe, it would be better if it is named as`login_url`, as in the reference it is  `test_url = '/test_url'`